hackathon/views.py

```diff
@@ -1,10 +1,61 @@
 from django.views.generic import ListView
+from django.shortcuts import render, get_object_or_404, redirect
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from datetime import datetime
 
-from .models import Hackathon
+from .models import Hackathon, HackAwardCategory, HackTeam, HackProject, HackProjectScore, HackProjectScoreCategory
+from .forms import HackathonForm
 
 
 class HackathonListView(ListView):
     """Renders a page with a list of Hackathons."""
     model = Hackathon
     ordering = ['-created']
     paginate_by = 8
+
+
+@login_required
+def create_event(request):
+    """ Allow users to create hackathon event """
+
+    if request.method == 'GET':
```

Comment:
I would put this in an else clause for the `if request.method == 'POST'` instead of having two serparate `if` statements.

Reply:
Again, this is my employer's preference, which is why I did it this way. I've changed that line to an else clause instead now.